metric-anomaly-detector/src/main/java/org/hypertrace/alert/engine/metric/anomaly/detector/MetricAnomalyDetector.java

```diff
@@ -57,6 +57,15 @@
     metricQueryBuilder = new MetricQueryBuilder(asClient);
   }
 
+  MetricAnomalyDetector(Config appConfig, AttributeServiceClient asClient, QueryServiceClient qsc) {
```

Comment:
call it `queryServiceClient ` only

Reply:
Another variable with same name exists

Reply:
which one?

Reply:
you just do `this.queryServiceClient = queryServiceClient`